es-bs-base/package.json

```diff
@@ -90,14 +87,14 @@
     "karma-qunit": "^4.1.2",
     "karma-sinon": "^1.0.5",
     "lockfile-lint": "^4.7.4",
-    "node-sass": "^7.0.1",
     "nodemon": "^2.0.15",
     "npm-run-all": "^4.1.5",
     "popper.js": "^1.16.1",
     "postcss": "^8.4.12",
     "postcss-cli": "^10.0.0",
     "qunit": "^2.18.2",
     "rollup": "^3.2.5",
+    "sass": "^1.66.1",
```

Comment:
*sass* (*dart-sass*) is the latest and most feature-complete compiler for sass adding `@use` and `@forward` support.